src/main/java/org/ibp/api/rest/study/StudyInstanceResource.java

```diff
@@ -64,9 +67,9 @@
 		return new ResponseEntity<>(studyInstances, HttpStatus.OK);
 	}
 
-	@ApiOperation(value = "Get study instance with basic metadata.",
-		notes = "Get study instance with basic metadata.")
-	@PreAuthorize("hasAnyAuthority('ADMIN','BREEDING_ACTIVITIES','MANAGE_STUDIES', 'INFORMATION_MANAGEMENT', 'BROWSE_STUDIES')")
```

Comment:
Same here, why did we remove the following authorities from seeing the instance: 'INFORMATION_MANAGEMENT', 'BROWSE_STUDIES'